public/fonts/icomoon.svg

```diff
@@ -4,11 +4,12 @@
+<glyph unicode="&#xe902;" glyph-name="sun-stroke" horiz-adv-x="1023" d="M512.533 460.637c70.565 0 128.125-57.496 128.125-128.125s-57.56-128.125-128.125-128.125-128.125 57.496-128.125 128.125 57.56 128.125 128.125 128.125zM512.533 588.762c-141.514 0-256.25-114.736-256.25-256.25s114.736-256.25 256.25-256.25c141.514 0 256.218 114.736 256.218 256.25s-114.704 256.25-256.218 256.25v0zM448.438 780.95c0 35.381 28.682 64.063 64.063 64.063s64.063-28.682 64.063-64.063c0-35.381-28.682-64.063-64.063-64.063s-64.063 28.682-64.063 64.063zM128.125 652.825c0 35.381 28.682 64.063 64.063 64.063s64.063-28.682 64.063-64.063c0-35.381-28.682-64.063-64.063-64.063s-64.063 28.682-64.063 64.063zM64.063 396.575c35.395 0 64.063-28.668 64.063-64.063 0-35.459-28.668-64.063-64.063-64.063s-64.063 28.604-64.063 64.063c0 35.395 28.668 64.063 64.063 64.063zM128.125 12.199c0 35.381 28.682 64.063 64.063 64.063s64.063-28.682 64.063-64.063c0-35.381-28.682-64.063-64.063-64.063s-64.063 28.682-64.063 64.063zM448.438-115.926c0 35.523 28.668 64.063 64.063 64.063 35.491 0 64.063-28.54 64.063-64.063 0-35.298-28.572-64.063-64.063-64.063-35.395 0-64.063 28.764-64.063 64.063zM768.751 12.199c0 35.381 28.682 64.063 64.063 64.063s64.063-28.682 64.063-64.063c0-35.381-28.682-64.063-64.063-64.063s-64.063 28.682-64.063 64.063zM960.938 268.45c-35.363 0-64.063 28.7-64.063 64.063 0 35.459 28.7 64.063 64.063 64.063s64.063-28.604 64.063-64.063c0-35.363-28.7-64.063-64.063-64.063zM768.751 652.825c0 35.381 28.682 64.063 64.063 64.063s64.063-28.682 64.063-64.063c0-35.381-28.682-64.063-64.063-64.063s-64.063 28.682-64.063 64.063z" />
```

Comment:
These glyphs seemed to have gained a lot of extra decimal places in their paths.  If we can prevent that, we should, unless it was intentional of course.